ingest/lmv1.go

```diff
@@ -0,0 +1,63 @@
+package ingest
+
+import (
+	"crypto/hmac"
+	"crypto/sha256"
+	"encoding/base64"
+	"encoding/hex"
+	"strconv"
+	"strings"
+	"time"
+)
+
+type lmv1Token struct {
```

Comment:
Anything in here can be public. Move anything to do with public api-token auth to some package like `ingest/apitoken` or better it's own base package `/apitoken`.